site/content/contribute/gsoc/_index.md

```diff
@@ -0,0 +1,223 @@
+
+### Spotify plugin
+
+There are tons of integrations that are potentially useful for our users. One that seems to be asked for quite a lot is an integration with Spotify. There are several features that we would like to see in such a plugin, but the development of this idea is open to the interests of the student:
+
+-   Share what the user is listening to in a pre-defined channel.
+-   Add a new section to the profile popover that shows the current song being played.
+-   Add player controls inside Mattermost: Location to be decided, but we would love to control what we are listening to without ever leaving Mattermost.
+-   Collective playlist with all the songs shared to a channel.
+
+This project is open to ideas that should be reflected in the student proposal.
+
+-   **Expected outcomes**: A new plugin to integrate Spotify with Mattermost.
+-   **Skills required**: React, Go
+-   **Difficulty**: Medium
+-   **Mentors**: Alejandro García Montoro
+
+### Quote plugin
+
+When we are writing in a busy thread, and we want to answer a specific post, there's an unwritten rule that says that you should prepend your post with a quote of that message, formatting it as a quote with the usual Markdown syntax. This is a manual process, and as such, it is sometimes skipped. But what if we could automate it?
+
+This project has that simple purpose: implement a new plugin that would add a new entry, `Quote`, to the post menu that would copy the corresponding post in the input box, properly formatted by prepending it with `> `. The plugin should never delete the current text in the input box, only adding this block of text.
+
+However, as of today, plugins are not able to modify the input textbox, so the first and most important task in this project would be to change the plugin API to add such a feature. These architectural changes are valuable in themselves, so a project that would only deliver this would be considered successful. However, if the student finishes this task before the coding period ends, the next task would be to actually implement the quote plugin, using the new functionality as a basis for it.
+
+-   **Expected outcomes**: The infrastructure changes for letting plugins modify the input box. If there is time, a new plugin that adds the quoting feature.
+-   **Skills required**: React
+-   **Difficulty**: Medium
+-   **Mentors**: Daniel Espino García, Alejandro García Montoro
+
+### Timed reminders on specific posts
+
+It's usual that we read an interesting message but don't have time to process it at the moment. Some of us use the "Mark as unread" feature or the TODO plugin for this, but it would be very valuable to have a feature that reminds us of that specific message in some time, generating a new notification in 30 minutes, one hour, or the next day.
```

Comment:
```suggestion
Often we read an interesting message but don't have time to process it at that moment. Some of us use the "Mark as unread" feature or the TODO plugin for this, but it would be valuable to have a feature that reminds us of a specific message after an elapsed period of time, such as after 30 minutes, one hour, or the next day.
```